library/errata_tool_product_version.py

```diff
@@ -303,6 +303,8 @@ def run_module():
         is_rhel_addon=dict(type='bool', required=True),
         push_targets=dict(type='list', required=True),
         brew_tags=dict(type='list', required=True),
+        use_quay_for_containers=dict(type="bool", default=False),
+        use_quay_for_containers_stage=dict(type="bool", default=False),
```

Comment:
I prefer us to avoid specifying a `default` value, so that Ansible will default to `None`. Later on, right before you call `ensure_product_version()`, if the value is indeed `None`, then drop it from params. Like this:

```
    # `use_quay_for_containers` is optional - don't modify the server-side value if the user has omitted it.
    if params['use_quay_for_containers'] is None:
        params.pop('use_quay_for_containers')
    if params['use_quay_for_containers_stage'] is None:
        params.pop('use_quay_for_containers_stage')
```

When we don't send the parameter(s) at all, we rely on the Errata Tool Rails' logic to choose reasonable defaults. This reduces the business logic in Ansible and keeps it where it belongs - with the ET developers. They're already choosing defaults in their web UI, and I want the Ansible experience to match the ET web UI experience so that it's easy for release engineers to move between Ansible and the web UI.